packages/trpc/server/routers/viewer/workflows/update.handler.ts

```diff
@@ -666,7 +714,7 @@ export const updateHandler = async ({ ctx, input }: UpdateOptions) => {
     });
   }
 
-  for (const eventTypeId of activeOn) {
+  for (const eventTypeId of activeOnWithChildren) {
```

Comment:
@leog I just pushed a fix for showing the SMS reminder field correctly. `activeOnWithChildren` should now include also children event types and `removedEventTypes` also has the children event types included now

Reply:
Looks good, queuing the merge, thanks @CarinaWolli!